router.go

```diff
@@ -485,3 +486,16 @@ func isPrivateSubnet(ipAddress net.IP) bool {
 	}
 	return false
 }
+
+func getContentSecurityPolicy() string {
+	policy := `default-src 'self';
+	script-src 'self' https://ajax.googleapis.com/ajax/libs/angularjs/1.4.5/angular.js https://cdnjs.cloudflare.com/ajax/libs/ng-csv/0.3.6/ng-csv.min.js https://cdn.polyfill.io/v2/polyfill.min.js https://edge.fullstory.com/s/fs.js;
```

Comment:
better to include the entire domain instead of one single script. also no need to include `https://`